### PR TITLE
new feature: implement tests abstraction with Suite

### PR DESCRIPTION
**Pull request by ivyoverflow**
#12 is closed.

### CODE REVIEW

File: book/internal/repository/suite.go [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+				Name:        "Concurrency in Go: Tools and Techniques for Developers",
+				DateOfIssue: "2017",
+				Author:      "Katherine Cox-Buday",
+				Description: `Concurrency can be notoriously difficult to get right, but fortunately, the Go open source programming
+				language makes working with concurrency tractable and even easy. If you’re a developer familiar with Go,
+				this practical book demonstrates best practices and patterns to help you incorporate concurrency into your systems.
+				Author Katherine Cox-Buday takes you step-by-step through the process.
+				You’ll understand how Go chooses to model concurrency, what issues arise from this model,
+				and how you can compose primitives within this model to solve problems.
+				Learn the skills and tooling you need to confidently write and implement concurrent systems of any size.`,
```
Comment (kiryalovik) [inline book/internal/repository/suite.go:54]: Not a blocker:
you don't need to use such big texts if it's not a test for corner cases. It's just will be easier to read with small texts

**Review by kiryalovik — APPROVED** [review]
Looks good!
Just resolve conflicts.
